src/components/memorygame/MemoryGameCards/index.jsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useEffect } from "react";
+import "./index.css";
+import ReactCardFlip from "react-card-flip";
+import { Col, Card, Modal } from "antd";
+import { useTranslation } from "react-i18next";
+import questionMark from "../../../images/memoryGameFrontImage.png";
+import memoryGameSuccessSymbol from "../../../images/memoryGameSuccessSymbol.svg";
+import cardInitialStateGenerator from "../cardInitialStateGenerator";
+
+const NUM_REQUIRED_MATCHES = 2;
+// game example: https://www.brightfocus.org/alzheimers/memory-games/match-pictures
+// instead of squares this game uses cards.
+// onClick card flips to show what is behind the card.
```

Comment:
onClick sounds like a function that is a prop to this component. "Card flips on click to show what is behind."

Reply:
PTAL